From 15627700fc3965feba38f9e0eb1bcf2f37186f37 Mon Sep 17 00:00:00 2001
From: Jianjun Zhu <jianjun.zhu@intel.com>
Date: Wed, 25 Nov 2020 22:17:07 +0800
Subject: [PATCH] Add owt_quic_transport to BUILD.gn.

---
 BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/BUILD.gn b/BUILD.gn
index c4f50576eaf4..65f013734085 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -88,6 +88,7 @@ group("gn_all") {
     "//ui/color:color_unittests",
     "//ui/gfx:gfx_unittests",
     "//url:url_unittests",
+    "//owt/quic_transport:owt_quic_transport",
   ]
 
   if (!is_component_build) {
-- 
2.17.1

